### DISCUSSION TITLE
Deploying project to Heroku

### DISCUSSION THREAD

**Discussion by manuelalferez** [manuelalferez/chatcus#36] [Q&A]
Hi @gitwikc, 

I am trying to deploy the subdirector the same way you indicated in #33. Could you teach me how did you deploy it last time?

![Captura de pantalla 2021-10-19 a las 12 24 41](https://user-images.githubusercontent.com/38152841/137891761-df5f807c-49e0-4104-a1c3-72613e33685e.png)

**Comment by manuelalferez** [comment]
I made the decision to move to Google Cloud. Now everything is mounted on a Docker machine, and it runs as we normally do in local: npm start. Previous configuration apart from [Dockerfile](https://github.com/manuelalferez/chatcus/blob/master/Dockerfile)

**Comment by gitwikc** [comment]
> Hi @gitwikc,
> 
> I am trying to deploy the subdirector the same way you indicated in #33. Could you teach me how did you deploy it last time?
> 
> ![Captura de pantalla 2021-10-19 a las 12 24 41](https://user-images.githubusercontent.com/38152841/137891761-df5f807c-49e0-4104-a1c3-72613e33685e.png)

Use the `git subtree` command from project directory:
`git subtree push --prefix server heroku master`
